packages/client-beta/src/apps/Soundboard/Soundboard.tsx

```diff
@@ -25,8 +26,12 @@ function Soundboard() {
     }
   };
 
-  function playXSoundLocally(url: string) {
-    XSoundsPlayService.play(url);
+  function playXSound(sound: XSound, event: MouseEvent) {
+    if (event.metaKey || event.altKey) {
+      XSoundsPlayService.play(sound.url);
```

Comment:
WDYT about removing `XSoundsPlayService` and moving its logic to `XSoundsPlayService` as `SoundboardService.playXSoundLocally` function?

Reply:
I'm not sure. `XSoundsPlayService` is all about reacting to the events and playing the sound locally. I'm not sure if this is the right place for actually playing the soundboard sound.

Reply:
So naming became a bit confusing. Why `XSoundsPlayService` is about locally playing any url? 
For me me it should be reversed. `XSoundsPlayService` sends request to backend, and `SoundsService` plays locally any sound (not only x?) from provided url.

Reply:
I see. I will try to propose something that makes more sense.